### PR TITLE
Inform developers how to set their custom HTTP status code for kernel.exception event.

### PR DESCRIPTION
**Pull request by gmponos**
Added a tip section that will inform users how to set their custom HTTP status code.

### CODE REVIEW

**Review by ndench — APPROVED** [review]
Thanks for the tip!

File: components/http_kernel.rst [hunk 1]
```diff
@@ -545,6 +545,11 @@ below for more details).
     When setting a response for the ``kernel.exception`` event, the propagation
     is stopped. This means listeners with lower priority won't be executed.
 
+.. note::
```
Comment (kunicmarko20) [inline components/http_kernel.rst]: maybe use `tip::` instead? :smile:

File: components/http_kernel.rst [hunk 2]
```diff
@@ -545,6 +545,11 @@ below for more details).
...
+
+    When setting a response for the ``kernel.exception`` event in order for the 
+    HttpKernel to use the Status Code of the response the :class:`Symfony\\Component\\HttpKernel\\Event\\GetResponseForExceptionEvent::allowCustomResponseCode` must be called inside your listener.
```
Comment (kunicmarko20) [inline components/http_kernel.rst]: instead of `:class:` probably `:method:`?

**Review by kunicmarko20 — COMMENTED** [review]
Just a few minor comments, thank you for helping!

File: components/http_kernel.rst (hunk 1 above)
Comment (gmponos) [reply components/http_kernel.rst]: check now.

File: components/http_kernel.rst (hunk 2 above)
Comment (gmponos) [reply components/http_kernel.rst]: check now.

File: components/http_kernel.rst [hunk 3]
```diff
@@ -545,6 +545,11 @@ below for more details).
     When setting a response for the ``kernel.exception`` event, the propagation
     is stopped. This means listeners with lower priority won't be executed.
 
+.. tip::
+
+    When setting a response for the ``kernel.exception`` event in order for the 
+    HttpKernel to use the Status Code of the response the :method:`Symfony\\Component\\HttpKernel\\Event\\GetResponseForExceptionEvent::allowCustomResponseCode` must be called inside your listener.
```
Comment (kunicmarko20) [inline components/http_kernel.rst]: `HttpKernel` should probably be in double backticks, sorry didn't notice it the first time.

Comment (gmponos) [reply components/http_kernel.rst]: >  should probably 

are you sure? Checking on the same document other links have this form. Meaning with single quotes.

**Comment by kunicmarko20** [comment]
This is already mentioned in https://github.com/symfony/symfony-docs/blob/master/reference/events.rst

File: components/http_kernel.rst (hunk 3 above)
Comment (kunicmarko20) [reply components/http_kernel.rst]: Let's wait for someone else to reply

**Pull request closed without merging**